packages/authgear-react-native/index.d.ts

```diff
@@ -928,56 +933,94 @@ export declare class ReactNativeContainer {
    */
   configure(options: ConfigureOptions): Promise<void>;
   /**
-   * Authenticate the end user via the web.
+   * authenticate() starts the authentication process in a webview.
+   * After authentication, the webview will be closed and the user is logged in.
+   *
+   * You can refer to {@link AuthenticateOptions} for more customization.
    *
    * @public
    */
   authenticate(options: AuthenticateOptions): Promise<AuthenticateResult>;
   /**
+   * changePassword() opens the settings page in a webview for the user to change their password.
+   * After changing the password, the webview will be closed.
+   *
+   * You can refer to {@link SettingsActionOptions} for more customization.
+   *
    * @public
    */
   changePassword(options: SettingsActionOptions): Promise<void>;
   /**
+   * deleteAccount() opens the settings page in a webview for the user to delete their account.
+   * After deletion, the webview will be closed and the user is logged out.
+   *
+   * You can refer to {@link SettingsActionOptions} for more customization.
+   *
    * @public
    */
   deleteAccount(options: SettingsActionOptions): Promise<void>;
   /**
-   * Reauthenticate the end user via biometric or the web.
+   * reauthenticate() starts the reauthentication process via biometric or in the webview.
    *
    * If biometricOptions is given, biometric is used when possible.
    *
+   * You can refer to {@link ReauthenticateOptions} and {@link BiometricOptions} for more customization.
+   *
    * @public
    */
   reauthenticate(
     options: ReauthenticateOptions,
     biometricOptions?: BiometricOptions
   ): Promise<ReauthenticateResult>;
+  /**
+   * Open Authgear pages. Currently only settings pages are available.
+   *
+   * You can refer to {@link SettingOptions} for more customization.
+   *
+   * @public
+   */
   open(page: Page, options?: SettingOptions): Promise<void>;
   /**
-   * Logout.
+   * logout() starts the logout process in the background.
+   * After logging out, the access token will be revoked and cleared from the token storage.

```

Comment:
It is not the access token will be revoked and cleared from the token storage. We never store access tokens in the token storage.

> logout() signs out the user without showing any UI. The refresh token will be cleared.
> The biometric authentication is kept, so the user can authenticateBiometric again.